airgun/entities/hardware_model.py

```diff
@@ -48,7 +48,7 @@ def delete(self, entity_name, err_message=''):
         view.table.row(name=entity_name)['Actions'].widget.click()
         view.delete_dialog.confirm()
         if err_message:
-            view.flash.assert_message(err_message)
+            view.flash.assert_message(f"Danger alert: {err_message}")
```

Comment:
@peterdragun Is this change applicable for 6.11.z as well? If not, we can add 6.12 label to this PR.

Reply:
Yeah, it is, as PF4 notifications are also presented in 6.11. As mentioned changes were tested on both 6.12 and 6.11, to make sure that everything works.